Mathlib/Logic/Equiv/LocalEquiv.lean

```diff
@@ -34,7 +34,7 @@ As for equivs, we register a coercion to functions and use it in our simp normal
 `LocalEquiv.refl`    : the identity local equiv
 `LocalEquiv.ofSet`  : the identity on a set `s`
 `EqOnSource`        : equivalence relation describing the "right" notion of equality for local
-                        equivs (see below in implementation notes)
+                      equivs (see below in implementation notes)
```

Comment:
This whole section is a disaster on the online docs:

![image](https://github.com/leanprover-community/mathlib4/assets/425260/ba37a184-6aa0-46a8-a06e-5b5a42b8b8b3)

Can you change this to a bulleted list, and drop the column alignment? (either here, or in a separate PR if you think it's out of scope)

Reply:
Fixed in #8122

Reply:
Thank you!